src/Uno.UI.RemoteControl.Host/RemoteControlServer.cs

```diff
@@ -240,6 +299,24 @@ public void Dispose()
 			{
 				processor.Value.Dispose();
 			}
+
+			// Unload any AssemblyLoadContexts not being used by any current connection
+			foreach (var appId in _appInstanceIds)
```

Comment:
We're enumerating a shared resource, we should move the `_appInstanceIds` to a ConcurrentDictionary.

Reply:
the _appInstanceIds are unique to the Server instance. It's the _loadContext that's shared so I'll make that concurrent.

I'm also open to renaming these variables if you have better suggestions.